tools/c7n_azure/tests_azure/templates/tfprovision.sh

```diff
@@ -0,0 +1,124 @@
+#!/bin/bash
+IFS=$'\n\t'
+# IFS new value is less likely to cause confusing bugs when looping arrays or arguments (e.g. $@)
+
```

Comment:
set -ex  with OPIPE fail, ie ideally a failed shell command here needs to be trapped or fail the shell script instead of continuing blithely.

Reply:
https://vaneyckt.io/posts/safer_bash_scripts_with_set_euxo_pipefail/